src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -2866,6 +2887,30 @@ def test_api_video_update_detail_token_user_upload_state(self):
         video.refresh_from_db()
         self.assertEqual(video.upload_state, PENDING)
 
+    def test_api_video_update_detail_token_user_join_mode(self):
+        """Token users should be able to update the join mode of their video through the API."""
+        video = factories.VideoFactory(description="my description")
```

Comment:
Just for reassurance, I would add a self.assertEqual(video.join_mode, OPEN) or force the initialization of the value join_mode=CLOSED before the request.
We can't be sure that the request updated anything, maybe it was already set at CLOSED with its default value, it could be the case if the default value changes in time. WDYT